worlds/sm64ex/Options.py

```diff
@@ -96,25 +106,43 @@ class CompletionType(Choice):
 
 
 class ProgressiveKeys(DefaultOnToggle):
-    """Keys will first grant you access to the Basement, then to the Secound Floor"""
+    """Keys will first grant you access to the Basement, then to the Second Floor"""
     display_name = "Progressive Keys"
 
+class StrictMoveRequirements(DefaultOnToggle):
+    """If disabled, Stars that expect certain moves may have to be acquired without them. Only makes a difference
+    if Move Randomization is enabled"""
+    display_name = "Strict Move Requirements"
+
+def getMoveRandomizerOption(action: str):
+    class MoveRandomizerOption(Toggle):
+        """Mario is unable to perform this action until a corresponding item is picked up.
+        If Mario 64 is built with an alternative "nomoverando" branch, this option will not affect Mario."""
```

Comment:
I'd rather this say something like `This option is incompatible with builds using a 'nomoverando' branch`.
The `nomoverando` branch prints huge warnings on screen if they attempt to load a game where move rando is enabled, its not like its a cheat code to have it not affect mario (Except if they can ignore that I guess).

Reply:
Done.

https://github.com/ArchipelagoMW/Archipelago/pull/2569/commits/1cab3efe156f91109f031006808fab88bfe0490f